Setup/UpgradeSchema.php

```diff
@@ -63,6 +64,10 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
             $this->updateSchemaVersion207($setup);
         }
 
+        if (version_compare($context->getVersion(), '2.2.1', '<')) {
```

Comment:
Should we change this to 2.3.0?

Reply:
no not needed this will be executed as release before was 2.2.0.